main.tf

```diff
@@ -119,9 +119,38 @@ resource "aws_iam_role" "tasks" {
   assume_role_policy = file("${path.module}/policies/ecs-task-execution-role.json")
 }
 
+data "aws_iam_policy_document" "tasks" {
+  statement {
+    sid = "1"
+    actions = [
+      "ecr:GetAuthorizationToken",
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:BatchGetImage",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "ssm:GetParameter*"
+    ]
+    resources = ["${var.aws_ssm_allowed_parameters_arn}"]
+  }
+
+  statement {
+    actions = [
+      "ssm:DescribeParameters",
```

Comment:
Only `ssm:GetParameters` is required as described here -> https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html